bindings/python/tests/bindings/test_decoders.py

```diff
@@ -169,3 +169,27 @@ def test_can_modify(self):
 
         decoder.cleanup = False
         assert decoder.cleanup == False
+
+
+class TestSequenceDecoder:
+    def test_instantiate(self):
+        assert Sequence([]) is not None
+        assert Sequence([CTC()]) is not None
+        assert isinstance(Sequence([]), Decoder)
+        assert isinstance(Sequence([]), Sequence)
+        serialized = pickle.dumps(Sequence([]))
+        assert isinstance(pickle.loads(serialized), Sequence)
+
+    def test_decoding(self):
+        decoder = Sequence([CTC(), Metaspace()])
+        initial = ["▁", "▁", "H", "H", "i", "i", "▁", "y", "o", "u"]
+        expected = [
+            "",
```

Comment:
Shouldn't empty strings be discarded ?

Reply:
It might, but I wouldn't do it pre-emptively. Simpler code is usually more correct, so I'll keep this as-is, but if it starts causing issues we can change this easily